### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 19

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#195] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152253852-fa0280ab-e93e-4124-b1de-a4568584d511.png)

**Comment by Exyss** [comment]
1. $\binom{30}{3} \binom{60}{7}$
2. $\binom{3}{3} \binom{30}{7}$
3. $\binom{2}{1} \binom{28}{2} \binom{60}{7}$
4.
	**Per tipizzazione:**
		- $AlmenoUnAlfa = UnAlfa + DueAlfa = \binom{2}{1} \binom{28}{2} \binom{60}{7} + \binom{2}{2} \binom{28}{1} \binom{60}{7}$
		
	**Per complemento:**
		- $AlmenoUnAlfa = Totali - NessunAlfa = \binom{30}{3} \binom{60}{7} - \binom{28}{3} \binom{60}{7}$

**Comment by AxnNxs** [comment]
Stessi risultati di @Exyss
